uni/unidoc/buildIPLdoc.sh

```diff
@@ -2,17 +2,19 @@
 
 # Build and install the IPL documentation
 
-SBASE=/opt/unicon/ipl
-TBASE=/var/www/html/unicon/ipl
+# You must reset UBASE for your system's location of Unicon.
+UBASE=/opt/unicon
```

Comment:
Steve,  the configure scripts already gives you this location. See uni/ide/Makefile as an example: 

```
BASE=../..
include $(BASE)/Makedefs.uni
```

Makedefs.uni should provide any of such system dependent variables. If you need something that isn't there please let me know and I can add it. The end user shouldn't need to edit files individually.

Reply:
Thanks.  Makefile.uni,however, isn't sourceable by a script, just Makefiles.  I've been reluctant to force the use of
the scripts into the make process because running both adds about 24MB of html files into the doc hierarchy.  If
folks are okay with that I can merge there calls into the Makefile.  However, simply setting UBASE to a relative path
will also work in the build scripts.  I'll do that for now.

Reply:
We do have:
docdir=${datarootdir}/doc/${PACKAGE_TARNAME}
mandir=${datarootdir}/man
datarootdir = ${prefix}/share

But we can add one from html generated  docs. We can add a configure option to generate html docs with a configurable target install location for such files. Would that be a good option ?

Reply:
I like that.  One thing I'm trying to decide is how much source to process in the html generation.  Right now it's
just some of the directories in IPL (19MB of html) and uni/lib (5MB).  It's tempting to expand it further but that's
a lot of space...

Reply:
Would you like me to update the Makefile with the missing lines and switch unicon to $(UNICON)?

Reply:
That would be better probably. This way it aligns with  how the rest of the libraries are built.